server/domain/status/adapter/TravisCI.js

```diff
@@ -0,0 +1,43 @@
+const Status = require('../Status');
+const StatusManager = require('../StatusManager');
+const { parse } = require('querystring');
+const gravatar = require('gravatar');
+
+class StatusAdapterTravisCI {
+    processWebHook(data) {
+        data = JSON.parse(data);
+
+        // Do not process pull requests
+        if (data.type === 'pull_request') {
+            return;
+        }
+
+        Status.createStatus({
+            key: data.id.toString(),
```

Comment:
This is too unique now, you want to overwrite it. Most ideal key would be:

`travis-ci-${project.name}-${project.branch}` and make sure it's `[a-Z0-9-_]`